entries/g/gm.com.json

```
@@ -0,0 +1,22 @@
+{
+  "General Motors": {
+    "domain": "gm.com",
+    "additional-domains": [
+      "chevrolet.com",
+      "buick.com",
+      "gmc.com",
+      "cadillac.com"
+    ],
+    "img": "gm.com.png",
+    "tfa": [
+      "sms",
+      "email",
+      "totp"
+    ],
+    "notes": "Automatically turned on for email. You are only allowed to have one method turned on at a time.",
+    "categories": [
+      "transport",
+      "other"
```

```suggestion
      "transport"
```

I would place it only in `transport`

---

Thanks @hkamran80. I have updated the category to `transport` only.